### ISSUE TITLE
✅ ⛔ 🔗 🐞 Bug Report: Buttons in popups are autofocusing and there appears to be no way to prevent this. (Duplicate of #21409)

### ISSUE THREAD

**Issue by andrewj1313** [elementor/elementor#21466]
### Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

### Description

I just pushed my website from Dev to Live in WPEngine and then updated my Elementor Pro to the lates version (3.11.2 from 3.10.3). I have a popup that appears at the bottom of every page and after the update, the first button in the popup is automatically focused. I have checked my Dev environment before the Elementor update and this is not an issue. I then ran the plugin update on my Dev and the same issue occurred so I have reverted back to 3.10.3 and everything functions as expected/wanted.

I saw a similar thread here: https://github.com/elementor/elementor/issues/21360 where it was determined that this issue is not Elementor, but in this case, it clearly is. Elementor is the only plugin that was updated and when it is, the issue appears.

I can't imagine this was added in as an ADA feature as it would seem to be a violation of ADA/WCAG behavior, so I'm assuming it's a bug in the logic somewhere OR it was added as an option that is not available for users to turn off.

### Steps to reproduce

1. Install/update to Elementor 3.11.2
2. Create a popup and have it appear on any page and include an actionable item, like a button
3. The button will be autofocused when the popup appears

### Isolating the problem

- [X] This bug happens with only Elementor plugin active (and Elementor Pro).
- [X] This bug happens with a Blank WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [X] I can reproduce this bug consistently following the steps above.

### System Info

<details>
<summary>System Info </summary>

````txt
					
== Server Environment ==
	Operating System: Linux
	Software: Apache
	MySQL version: Percona Server (GPL), Release '43', Revision 'c1b94a6cfd7' v5.7.40-43
	PHP Version: 7.4.33
	PHP Memory Limit: 512M
	PHP Max Input Vars: 10000
	PHP Max Post Size: 100M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 6.1.1
	Site URL: https://www.themilleventcenter.com
	Home URL: https://www.themilleventcenter.com
	WP Multisite: No
	Max Upload Size: 50 MB
	Memory limit: 40M
	Max Memory limit: 512M
	Permalink Structure: /%year%/%monthnum%/%day%/%postname%/
	Language: en-US
	Timezone: America/New_York
	Admin Email: lorena@themilleventcenter.com
	Debug Mode: Inactive

== Theme ==
	Name: PPA Astra Child
	Version: 1.0.0
	Author: PPA Agency
	Child Theme: Yes
	Parent Theme Name: Astra
	Parent Theme Version: 4.0.2
	Parent Theme Author: Brainstorm Force

== User ==
	Role: administrator
	WP Profile lang: en_US
	User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/110.0.0.0 Safari/537.36

== Active Plugins ==
	Advanced Custom Fields PRO
		Version: 6.0.7
		Author: WP Engine

	Astra Pro
		Version: 4.0.1
		Author: Brainstorm Force

	BugHerd
		Version: 1.0.4
		Author: BugHerd

	Classic Editor
		Version: 1.6.2
		Author: WordPress Contributors

	CMB2
		Version: 2.10.1
		Author: CMB2 team

	Custom Post Type UI
		Version: 1.13.4
		Author: WebDevStudios

	Easy Watermark
		Version: 1.0.11
		Author: BracketSpace

	Ele Custom Skin
		Version: 3.1.7
		Author: Dudaster.com

	Ele Custom Skin Pro
		Version: 3.2.4
		Author: Dudaster.com

	Elementor
		Version: 3.11.2
		Author: Elementor.com

	Elementor Pro
		Version: 3.11.2
		Author: Elementor.com

	email subscription popup
		Version: 1.2.16
		Author: I Thirteen Web Solution

	Header and Footer Scripts
		Version: 2.2.1
		Author: Digital Liberation

	Make Column Clickable Elementor
		Version: 1.4.0
		Author: Fernando Acosta

	Ninja Forms
		Version: 3.6.19
		Author: Saturday Drive

	Shortcodes Ultimate
		Version: 5.12.8
		Author: Vova Anokhin

	Smush
		Version: 3.12.5
		Author: WPMU DEV

	Ultimate Addons for Elementor
		Version: 1.36.14
		Author: Brainstorm Force

	VS Event List
		Version: 15.7
		Author: Guido

	WP Mail SMTP
		Version: 3.7.0
		Author: WPForms

	Yoast SEO
		Version: 20.1
		Author: Team Yoast


== Must-Use Plugins ==
	Force Strong Passwords - WPE Edition
		Version: 1.8.0
		Author: Jason Cosper

	WP Engine Cache Plugin
		Version: 1.1.2
		Author: WP Engine

	WP Engine Seamless Login Plugin
		Version: 1.6.0
		Author: WP Engine

	WP Engine Security Auditor
		Version: 1.0.10
		Author: wpengine

	WP Engine System
		Version: 5.0.1
		Author: WP Engine


== Elements Usage ==
	
	footer : 1
		divider : 2
		heading : 6
		social-icons : 1
	header : 1
		icon : 1
		nav-menu : 1
		theme-site-logo : 1
	loop : 1
		heading : 1
		icon : 1
		theme-post-content : 1
		theme-post-title : 1
	popup : 2
		button : 2
		uael-nav-menu : 1
	section : 1
		divider : 1
		heading : 1
	widget : 0
		divider : 2
	wp-post : 0
		divider : 2
		heading : 3
		image-box : 3
		posts : 1
		text-editor : 1
	wp-page : 9
		accordion : 1
		button : 3
		divider : 19
		heading : 33
		icon-box : 3
		icon-list : 8
		image : 1
		posts : 1
		shortcode : 2
		text-editor : 12
		uael-image-gallery : 8
		uael-price-table : 4


== Settings ==
	
	cpt_support: post, page
	allow_tracking: yes
	unfiltered_files_upload: 1
	font_display: swap
	load_fa4_shim: yes


== Features ==
	Custom Fonts: 0
	Custom Icons: 0

== Integrations ==
	


== Elementor Experiments ==
	Optimized DOM Output: Inactive by default
	Improved Asset Loading: Inactive by default
	Improved CSS Loading: Inactive by default
	Inline Font Icons: Inactive by default
	Accessibility Improvements: Inactive by default
	Additional Custom Breakpoints: Inactive by default
	admin_menu_rearrangement: Inactive by default
	Flexbox Container: Inactive by default
	Upgrade Swiper Library: Inactive by default
	Default to New Theme Builder: Active by default
	Editor V2: Inactive by default
	Landing Pages: Active by default
	Nested Elements: Inactive by default
	Lazy Load Background Images: Inactive by default
	Save as Default: Active by default
	Page Transitions: Active by default
	Notes: Active by default
	Loop: Active by default
	Form Submissions: Active by default
	Scroll Snap: Active by default
	Mega Menu: Inactive by default


== Log ==
	
Log: showing 20 of 352023-01-30 10:29:11 [info] Elementor/Upgrades - _v_3_0_0_move_default_typography_to_kit Finished 
2023-01-30 10:29:11 [info] Elementor/Upgrades - _v_3_2_0_migrate_breakpoints_to_new_system Start  
2023-01-30 10:29:11 [info] Elementor/Upgrades - _v_3_2_0_migrate_breakpoints_to_new_system Finished 
2023-01-30 10:29:11 [info] Elementor/Upgrades - _v_3_4_8_fix_font_awesome_default_value_from_1_to_yes Start  
2023-01-30 10:29:11 [info] Elementor/Upgrades - _v_3_4_8_fix_font_awesome_default_value_from_1_to_yes Finished 
2023-01-30 10:29:11 [info] Elementor/Upgrades - _v_3_5_0_remove_old_elementor_scheme Start  
2023-01-30 10:29:11 [info] Elementor/Upgrades - _v_3_5_0_remove_old_elementor_scheme Finished 
2023-01-30 10:29:11 [info] Elementor/Upgrades - _v_3_8_0_fix_php8_image_custom_size Start  
2023-01-30 10:29:11 [info] Elementor/Upgrades - _v_3_8_0_fix_php8_image_custom_size Finished 
2023-01-30 10:29:11 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.10.1',
  'to' => '3.10.2',
)]
2023-02-23 08:35:09 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.10.2',
  'to' => '3.11.2',
)]
2023-02-23 08:35:09 [info] elementor::elementor_updater Started 
2023-02-23 08:35:09 [info] Elementor/Upgrades - _on_each_version Start  
2023-02-23 08:35:10 [info] Elementor/Upgrades - _on_each_version Finished 
2023-02-23 08:35:10 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.10.2',
  'to' => '3.11.2',
)]
2023-02-25 10:02:32 [info] elementor-pro::elementor_pro_updater Started 
2023-02-25 10:02:32 [info] Elementor Pro/Upgrades - _on_each_version Start  
2023-02-25 10:02:32 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2023-02-25 10:02:32 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.10.3',
  'to' => '3.11.2',
)]
2023-02-25 10:02:32 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.10.3',
  'to' => '3.11.2',
)]

JS: showing 6 of 6JS: 2023-01-30 15:41:33 [error X 14][https://milleventdev.wpengine.com/wp-includes/js/jquery/jquery.min.js?ver=3.6.1:2:51824] Cannot read properties of undefined (reading 'replace') 
JS: 2023-01-30 16:16:56 [error X 10][https://milleventdev.wpengine.com/wp-content/plugins/ele-custom-skin-pro/assets/js/ecspro.js?ver=3.2.4:172:17] Cannot read properties of undefined (reading 'params') 
JS: 2023-01-31 21:09:17 [error X 1][https://milleventdev.wpengine.com/wp-content/plugins/elementor/assets/js/common-modules.min.js?ver=3.10.2:2:52931] Model.isValidChild() should be implemented, please provide 'isValidChild' functionality. 
JS: 2023-02-01 14:13:12 [error X 51][https://milleventdev.wpengine.com/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.10.2:3:782338] Cannot read properties of undefined (reading 'localized') 
JS: 2023-02-01 15:31:18 [error X 2][https://milleventdev.wpengine.com/wp-content/plugins/elementor/assets/js/new-template.min.js?ver=3.10.2:2:2932] elementor_new_template_form_controls is not defined 
JS: 2023-02-14 00:58:06 [error X 2][https://milleventdev.wpengine.com/wp-content/plugins/elementor/assets/js/frontend-modules.min.js?ver=3.10.2:2:8985] Cannot read properties of null (reading 'data') 

PHP: showing 2 of 2PHP: 2023-01-30 10:41:52 [notice X 5][/nas/content/live/milleventdev/wp-content/plugins/elementor/core/common/modules/ajax/module.php::172] Undefined index: data [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-01-30 10:56:05 [notice X 1][/nas/content/live/milleventdev/wp-content/plugins/elementor-pro/modules/dynamic-tags/acf/tags/acf-text.php::33] Undefined offset: 1 [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]



== Elementor - Compatibility Tag ==
	
	Elementor Pro: Compatibility not specified
	Make Column Clickable Elementor: Compatibility not specified
	Ultimate Addons for Elementor: Compatibility not specified

== Elementor Pro - Compatibility Tag ==

````
</details>

**Comment by nicholaszein** [comment]
Hello @andrewj1313,

Thank you for reaching out! 🙏 

### ⛔ While reviewing your report we concluded this is not an issue caused by Elementor. Here's what we found out:
- This is indeed a new feature, which is not working correctly, but can be turned on.
- The issue you mentioned was not describing the error correctly, and the poster was reluctant to share the required information, making it virtually impossible to validate it. Fortunately someone else was able to make a proper bug report, allowing us to understand what was being reported without any ambiguity. 

### ✅ What you can do to solve your issue:
A fix is under way, but meanwhile you can do the following:
1. Edit the Pop-up with Elementor.
2. Go to the Document Settings (⚙ gear icon at the bottom left corner of the widget panel). 
3. Open the Advanced tab and disable the new option called "Accessible Navigation".

### 👍 For now, that's all we can help with. 

----

### 🔗 🐞 This issue is a Duplicate of #21409

Which is on the same subject. 

For this reason, I'm closing this thread and 🔃 **Merging** it with the original, so we can have all the discussions in one location, and deal with it more efficiently.

Feel free to continue the conversation there. 😉 

### 🔍 Please, in the future, remember to check if there isn't any thread similar to yours already opened. 

Kind regards

**Comment by nicholaszein** [comment]
Hello @andrewj1313.

We have great news! 🙌 

### 📢 We're happy to announce that the issue you raised was resolved in Elementor Pro v3.17.0! 🥳 

The **`Accessible Navigation`** option in Popups exists to assist users in navigating the content inside the Popup. It's a crucial feature for a more accessible website.

Previously, users experienced a navigation challenge where the focus, upon triggering a Popup, would land on a random first focusable element like a button or a link. This often led to confusion, especially for users relying on assistive technologies, as the focus could be on an element located at the end of the Popup.

To enhance user experience, we've implemented a crucial change. Now, when a modal opens, the focus immediately shifts to the close button. This adjustment ensures a more intuitive and accessible interaction, allowing users, particularly those with disabilities, to easily locate and use the close button if they wish to exit the Popup. 

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
